README.md

```diff
@@ -25,12 +25,29 @@
  1. `cd ~/catkin_ws`
  2. `catkin_make`
 
- ## Running
+ ## Enable Hot Swapping
```

Comment:
Might want to make a note somewhere in this section that this step only has to be done once per computer